bangazonapi/models/product.py

```diff
@@ -62,8 +62,11 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
-        return avg
+            try:
+                avg = total_rating / len(ratings)
+            except:
+                avg = 0
+                return avg
```

Comment:
I believe this `return` should be indented one less time. That way `avg` always returns when it does and doesn't have a nonzero value.

Reply:
@mitchellblom done and re-pushed